src/app/public/modules/colorpicker/colorpicker.component.ts

```diff
@@ -76,22 +76,50 @@ import {
 
 let componentIdIndex = 0;
 
+/**
+ * Provides a SKY UX-themed replacement for the HTML `input` element with `type="color"`.
+ * The value that users select is driven through the `ngModel` attribute specified on
+ * the `input` element.
+ */
 @Component({
   selector: 'sky-colorpicker',
   templateUrl: './colorpicker.component.html',
   styleUrls: ['./colorpicker.component.scss']
 })
 
 export class SkyColorpickerComponent implements OnInit, OnDestroy {
+  /**
+   * Fires when users select a color in the colorpicker. Both events
+   * return an object with the following properties:
+   * ```{
+   * hslaText: string
+   * rgbaText: string
+   * cmykText: string
+   * hsva: { hue:  number, saturation: number, value: number,      alpha: number}
+   * rgba: { red:  number, green: number,      blue: number,       alpha: number}
+   * hsla: { hue:  number, saturation: number, lightness: number,  alpha: number}
+   * cmyk: { cyan: number, magenta: number,    yellow: number,     key: number  }
+   * hex:  string
+   * }```
+   */
   @Output()
   public selectedColorChanged = new EventEmitter<SkyColorpickerOutput>();
 
+  /**
+   * Fires when users select Apply in the colorpicker to apply a color.
```

Comment:
In the current docs, Apply is bold. Should we make it bold here or should we just leave it as is?

Reply:
Made it bold